client/router_create.go

```diff
@@ -653,10 +653,14 @@ func (cli *VanClient) GetRouterSpecFromOpts(options types.SiteConfigSpec, siteId
 				controllerHosts = append(controllerHosts, controllerIngressHost)
 			}
 		}
+		siteServerSubject := types.TransportServiceName
+		if options.IsIngressNodePort() {
+			siteServerSubject = options.GetRouterIngressHost()
```

Comment:
I think we need to check the length of the host name here, and only use it as the subject if it is less than 64 bytes.

Reply:
Ok, I will include that extra length validation.